cmake/install-rules.cmake

```diff
@@ -0,0 +1,57 @@
+# Copyright Microsoft Corporation.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+include(CMakePackageConfigHelpers)
+include(GNUInstallDirs)
+
+if(BUILD_PRINTER)
+  install(TARGETS ifc-printer DESTINATION . COMPONENT Printer)
```

Comment:
Did you choose `.` for any particular reason? I am changing it to `bin` in my env so I don't need to change my `PATH` if, for example, I install it with prefix `/usr/local`

ie: I think it's better to have it in `/usr/local/bin/ifc-printer` than just `/usr/local/ifc-printer`

Reply:
> I think it's better to have it in `/usr/local/bin/ifc-printer` than just `/usr/local/ifc-printer`

Agreed.  I should have spotted that during review.